.github/workflows/build.yml

```diff
@@ -92,7 +92,20 @@ jobs:
           -DOBS_FRONTEND_LIB:STRING=$GITHUB_WORKSPACE/obs-studio/build/UI/obs-frontend-api/libobs-frontend-api.dylib \
           -DQTDIR:STRING=/tmp/obsdeps ..
 
-        make -j
+        make -j obs-plugin
+
+    - name: Build the DAL plugin
+      run: |
+        rm -rf obs-mac-virtualcam/build
+        mkdir obs-mac-virtualcam/build
+        cd obs-mac-virtualcam/build
+
+        cmake -DLIBOBS_INCLUDE_DIR:STRING=$GITHUB_WORKSPACE/obs-studio/libobs \
```

Comment:
This is what I'm worried about, don't these also need to be compiled arm64?

Reply:
nah, not for the dal plugin to work